### PR TITLE
update placement doc

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/stolostron/backlog/issues/27018
Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

**Comment by haoqing0110** [comment]
I'm not sure if it's good to put placement before clusterset, but I adjust it as a standalone chapter, to be the same level of clustersets, this structure is similar to [https://open-cluster-management.io/concepts/. ](https://open-cluster-management.io/concepts/).

And also remove the extensible scheduling section and update some of the content.  Feel free to leave your comments. 

/assign @qiujian16 @showeimer

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 1]
```diff
@@ -18,12 +18,19 @@ See the following information about how placement with managed clusters works:
...
-* `Placements` select from that working set by using label and claim selectors.
-+
-*Important:* `Placement` does not select a `ManagedCluster` if there is no `ManagedClusterSet` bound to the placement namespace.
+* `Placements` filter `ManagedClusters` from that working set by using label and claim selectors.
 
 * The placement of `ManagedClusters` can be controlled by using taints and tolerations. See xref:../cluster_lifecycle/taints_tolerations.adoc#taints-tolerations-managed[Using taints and tolerations to place managed clusters] for more information.
 
+* `Placements` rank the clusters by the soft requirements and select a subset among them.
+
+*Note:*
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: Notes: (plural)

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 2]
```diff
@@ -18,12 +18,19 @@ See the following information about how placement with managed clusters works:
...
 * The placement of `ManagedClusters` can be controlled by using taints and tolerations. See xref:../cluster_lifecycle/taints_tolerations.adoc#taints-tolerations-managed[Using taints and tolerations to place managed clusters] for more information.
 
+* `Placements` rank the clusters by the soft requirements and select a subset among them.
+
+*Note:* 
+
+* You need to bind at least one `ManagedClusterSet` to a namespace by creating a `ManagedClusterSetBinding` in that namespace. 
+* You need role-based access to `CREATE` on the virtual sub-resource of `managedclustersets/bind`.
+
+Next will use some examples to describe how to use the `Placement` API. Read the below content to get a general understanding of the API first.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: See the following examples of the Placement API.... 

Read the following API information..

consider combining, Chris can help with this. (we don't use "we" in doc or "below")

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 3]
```diff
@@ -38,44 +45,12 @@ If not specified, all `ManagedClusters` that meet the placement requirements are
...
-      A higher weight indicates that the prioritizer receives a higher weight in the cluster selection, while 0 weight indicates that the prioritizer is disabled. A negative weight indicates that it is one of the last ones selected.
-
-**Note:** The `configurations.name` file will be removed in v1beta1 and replaced by the `scoreCoordinate.builtIn` file. If both `name` and `scoreCoordinate.builtIn` are defined, the value in `scoreCoordinate.builtIn` is used to determine the selection.
-
-[#placement-binding]
-== Placement examples
-
-You need to bind at least one `ManagedClusterSet` to a namespace by creating a `ManagedClusterSetBinding` in that namespace. *Note:* You need role-based access to `CREATE` on the virtual sub-resource of `managedclustersets/bind`. See the following examples:
+[#placement-labelselector-claimSelector]
+== Placement LabelSelector/ClaimSelector
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: also lets separate this so we are not using /

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 4]
```diff
@@ -228,6 +274,38 @@ spec:
             scoreName: cpuratio
 ----
 
+Below are the explaination of each field under `prioritizerPolicy`:
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: again, below we cannot use (accessibility issue) and explanation needs fixed

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 5]
```diff
@@ -228,6 +274,38 @@ spec:
...
+
+      ***** `resourceName` defines the resource name of the `AddOnPlacementScore`. The placement prioritizer selects the `AddOnPlacementScore` custom resource by this name.
+
+      ***** `scoreName` defines the score name inside of the `AddOnPlacementScore`. `AddOnPlacementScore` contains a list of the score name and the score value. The `scoreName`, specifies the score to be used by the prioritizer.
+
+    *** `weight` defines the weight of prioritizer. The value must be in the range of [-10,10].
+      Each prioritizer calculates an integer score of a cluster in the range of [-100, 100]. The final score of a cluster is determined by the following formula `sum(weight * prioritizer_score)`.
+      A higher weight indicates that the prioritizer receives a higher weight in the cluster selection, while 0 weight indicates that the prioritizer is disabled. A negative weight indicates that it is one of the last ones selected.
+
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: This section needs a lot of work, I am very confused... too many bullets. Consider headers.

File: multicluster_engine/main.adoc [hunk 6]
```diff
@@ -57,9 +57,9 @@ include::cluster_lifecycle/managedclustersets_create.adoc[leveloffset=+4]
 include::cluster_lifecycle/managedclustersets_assign_role.adoc[leveloffset=+4]
 include::cluster_lifecycle/managedclustersetbinding_create.adoc[leveloffset=+4]
 include::cluster_lifecycle/managedclustersets_add_cluster.adoc[leveloffset=+4]
-include::cluster_lifecycle/placement_managed.adoc[leveloffset=+4]
 include::cluster_lifecycle/taints_tolerations.adoc[leveloffset=+4]
 include::cluster_lifecycle/managedclustersets_remove_cluster.adoc[leveloffset=+4]
+include::cluster_lifecycle/placement_managed.adoc[leveloffset=+3]
 include::cluster_lifecycle/cluster_pool_intro.adoc[leveloffset=+3]
```
Comment (swopebe) [inline multicluster_engine/main.adoc:70]: `managing_placement_intro.adoc,` then links to the tasks...this is one big long file that is running together. Please build this with modular doc.

@cadawson for this amount of build change, I recommend you check out the branch and fix it there...it is harder to fix in a PR, but we cannot merge as-is.

PRs from developers are great for small changes, not for large one like this. I recommend you guys change this issue to a doc issue, story-point it, and add it to the sprint to account for the time to format properly.

File: multicluster_engine/multicluster_engine_overview.adoc [hunk 7]
```diff
@@ -58,8 +58,8 @@
 ** xref:cluster_lifecycle/managedclustersets_assign_role.adoc#assign-role-clustersets[Assigning users or groups Role-Based Access Control permissions to your ManagedClusterSet]
 ** xref:cluster_lifecycle/managedclustersetbinding_create.adoc#creating-a-managedclustersetbinding[Creating a ManagedClusterSetBinding resource]
 ** xref:cluster_lifecycle/managedclustersets_add_cluster.adoc#adding-clusters-to-a-managedclusterset[Adding a cluster to a ManagedClusterSet]
-** xref:cluster_lifecycle/placement_managed.adoc#placement-managed[Using ManagedClusterSets with Placement]
 ** xref:cluster_lifecycle/managedclustersets_remove_cluster.adoc#removing-a-managed-cluster-from-a-managedclusterset[Removing a cluster from a ManagedClusterSet]
+* xref:cluster_lifecycle/placement_managed.adoc#placement-managed[Using ManagedClusterSets with Placement]
 * xref:cluster_lifecycle/cluster_pool_intro.adoc#managing-cluster-pools[Managing cluster pools (Technology Preview)]
```
Comment (swopebe) [inline multicluster_engine/multicluster_engine_overview.adoc]: I just took out sub topics from the overview.

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 8]
```diff
@@ -339,16 +417,4 @@ spec:
...
-
-[#extensible-scheduling]
-== Extensible scheduling
-
-In placement resource-based scheduling, sometimes the prioritizer needs more data than the default value provided by the `MananagedCluster` resource to calculate the score of the managed cluster. For example, schedule the clusters based on CPU or memory usage data of the clusters that are fetched obtained through a monitoring system.
-
-The API `AddOnPlacementScore` supports a more extensible way to schedule based on customized scores.
-
-* You can specify the score in the `placement.yaml` file to select clusters.
-* As a  score provider, a 3rd party controller can run on either the hub cluster or the managed cluster, to maintain the lifecycle of `AddOnPlacementScore` and update score into it.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc:352]: third-party

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 9]
```diff
@@ -228,6 +274,38 @@ spec:
             scoreName: cpuratio
 ----
 
+Below are the explaination of each field under `prioritizerPolicy`:
+  
+  ** `mode` is either `Exact`, `Additive`, or `""` where `""` is `Additive` by default.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: This seems like a "concept" file, so break it out as such with appropriate title and bulleted list or sections can help this become digestible.

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc (hunk 5 above)
Comment (swopebe) [reply multicluster_engine/cluster_lifecycle/placement_managed.adoc]: Lists need to be parallel but I am not sure what goes with what....

Definition list should look like:

***Term:*** Defintion 

Or you can write complete sentences:

_The term is defined as a word to describe such and such._

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc (hunk 1 above)
Comment (haoqing0110) [reply multicluster_engine/cluster_lifecycle/placement_managed.adoc]: Fixed.

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc (hunk 3 above)
Comment (haoqing0110) [reply multicluster_engine/cluster_lifecycle/placement_managed.adoc]: Fixed.

**Comment by haoqing0110** [comment]
@cadawson thanks for the reviewing, I just removed the details of API referring to the future update of https://github.com/stolostron/rhacm-docs/pull/4456

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 10]
```diff
@@ -162,6 +119,83 @@ spec:
...
+spec:
+  tolerations:
+    - key: gpu
+      value: "true"
+      operator: Equal
+----
+
+- Select `ManagedClusters` with matching taints for a specified period of time.
+
+The value of `TolerationSeconds` represents the length of time that the toleration tolerates the taint. It can be useful when a managed cluster is offline, users can make applications deployed on this cluster to be transferred to another available managed cluster after a tolerated time. The `TolerationSeconds` value can be useful by automatically transferring applications that are deployed on a cluster that goes offline to another managed cluster after a specified length of time.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: It can be useful 

please change to
Is useful (verb be unnecessary)

Comment (swopebe) [reply multicluster_engine/cluster_lifecycle/placement_managed.adoc]: can be useful by automatically transferring (not quite sure this correct), please make concise and active:

can automatically transfer

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc [hunk 11]
```diff
@@ -268,7 +302,7 @@ status:
 
 You might want to select managed clusters for your placements according to the status of the add-ons that are deployed on them. For example, you want to select a managed cluster for your placement only if there is a specific add-on that is enabled on the cluster. 
 
-You can do this by specifying the label for the add-on, as well as its status, if necessary, when you create the Placement. A label is automatically created on a `ManagedCluster` resource if an add-on is enabled on the cluster. The label is automatically removed if the add-on is disabled.
+You can do this by specifying the label for the add-on, as well as its status, when you create the Placement. A label is automatically created on a `ManagedCluster` resource if an add-on is enabled on the cluster. The label is automatically removed if the add-on is disabled.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/placement_managed.adoc]: @cadawson in all tech writing, let's strive for conciseness: You can do this by specifying 

changes

Specify (to follow basic writing for conciseness and active voice)

**Review by swopebe — APPROVED** [review]
Few changes recommended for the writer.

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc (hunk 11 above)
Comment (haoqing0110) [reply multicluster_engine/cluster_lifecycle/placement_managed.adoc]: Fixed.

File: multicluster_engine/cluster_lifecycle/placement_managed.adoc (hunk 10 above)
Comment (haoqing0110) [reply multicluster_engine/cluster_lifecycle/placement_managed.adoc]: Fixed.

**Pull request merged**